include/deal.II/base/vectorization.h

```diff
@@ -25,6 +25,7 @@
 #include <algorithm>
 #include <array>
 #include <cmath>
+#include <cstring>
```

Comment:
Do you actually need this include?

Reply:
I think it's needed for `memcpy` used below.